hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: minor
+
+This release add ``initialize`` decorator for stateful testing
+(originally discussed in :issue:`1216`). ``initialize`` act as a special rule
+that is only called once at the very beginning of each run of the state-machine.
```

Comment:
nit: This description doesn't quite work with multiple initialize rules. Maybe something closer to "each initialize rule is only called once, and all initialize rules will be called before any normal rules are called"?